src/test/java/uk/gov/companieshouse/officer/delta/processor/tranformer/OfficerTransformTest.java

```diff
@@ -32,9 +23,18 @@
 import uk.gov.companieshouse.officer.delta.processor.model.enums.RolesWithPre1992Appointment;
 
 import java.time.Instant;
+import java.util.Arrays;
 import java.util.stream.Stream;
 
+import static org.hamcrest.CoreMatchers.instanceOf;
```

Comment:
Minor point: looks like IDE's optimise imports has moved static imports after non-static imports. It's conventional for the static imports to come first. I'll mention it in the next dev. meeting.

Reply:
According to [this](https://stackoverflow.com/questions/13376299/static-import-statements-which-should-never-be-after-nonstatic-imports) it is purely a style choice. Our style guide follows googles which states w[e should have static imports](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing) first so I'll make this change.